script/fill_flagged_cache.rb

```diff
@@ -1,7 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
As mentioned in PR description, happy to bring this back if there's a different sequence y'all wanna follow for deploying this stuff. Just let me know!